src/pe_reports/report_generator.py

```diff
@@ -22,12 +39,28 @@ def load_template():
 
 def export_set(prs):
     """Export PowerPoint report set to output directory."""
-    pptx_out = "Customer_ID_Posture_Exposure.pptx"
-    prs.save(os.path.join(SAVE_PATH, pptx_out))
+    try:
+        pptx_out = "Customer_ID_Posture_Exposure.pptx"
+        prs.save(os.path.join("/output", pptx_out))
+    except OSError:
+        print("No output available.")
     return
 
 
+def generate_reports(data, data_dir, out_dir):
+    """Gather assets to produce reports."""
+    # TODO: build code to connect customer db, encrypt and embed pdf reports. Issue #7: https://github.com/cisagov/pe-reports/issues/7
+
+
 def main():
+    """Set up logging and call the pe_reprots function."""
```

Comment:
There is no function called `pe_reprots` in here:
```suggestion
    """Set up logging and call the generate_reports function."""
```

Reply:
Resolved in fb98057